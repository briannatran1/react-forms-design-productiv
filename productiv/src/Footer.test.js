import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer";

describe('Footer component', function () {
  test('renders without crashing', function () {
    render(<Footer />);
  });

  test('contains expected text', function () {
    const result = render(<Footer />);
    expect(result.queryByText(
      'Prødutïv is copyright ©2020 by Flüffy Data Enterprises, Inc.'))
      .toBeInTheDocument();
  });

  test('matches snapshot', function () {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});