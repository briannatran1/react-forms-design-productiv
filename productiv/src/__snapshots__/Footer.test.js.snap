// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="App-footer mt-5 text-muted"
  >
    <small>
      Prødutïv is copyright ©2020 by Flüffy Data Enterprises, Inc.
    </small>
  </footer>
</DocumentFragment>
`;
